integration/tools/validation/src/main/java/alluxio/cli/UnderFileSystemContractTest.java

```diff
@@ -338,4 +338,4 @@ public static void main(String[] args) throws Exception {
       test.run();
     }
   }
-}
+}
```

Comment:
newline

Reply:
I'll do another scanning to resolve all formatting/checkstyle issues. This commit contains the main registry logic, and tested the tools are all working fine. I'll do a more thorough test in a bit too.